src/python/gem5/resources/resource.py

```diff
@@ -1108,9 +1117,9 @@ def Resource(
     This function was created to maintain backwards compability for v21.1.0
```

Comment:
```suggestion
    This function was created to maintain backwards compatibility for v21.1.0
```
This is not related to the reformat but compatibility is misspelled as compability.